### PR TITLE
Add quansight logo to supporters

### PR DESCRIPTION
**Pull request by mrocklin**
In particular this is due to their generously funding James Bourbeau to
maintain the project.

Is there a better location for the image?

cc @jrbourbeau @rgommers

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -451,6 +451,10 @@ <h2>Supported By</h2>
             <div class="col supporter">
               <a href="https://www.janelia.org"><img src="_images/HHMI_Janelia_Color.png"></a>
             </div>
+            <div class="col supporter">
+              <a href="https://quansight.com"><img
+                src="https://static.wixstatic.com/media/095d2c_2508c560e87d436ea00357abc404cf1d~mv2.png/v1/crop/x_0,y_9,w_915,h_329/fill/w_310,h_107,al_c,q_80,usm_0.66_1.00_0.01/095d2c_2508c560e87d436ea00357abc404cf1d~mv2.webp"></a>
```
Comment (rgommers) [inline index.html]: I would not rely on that link being stable. Why not stick an SVG image of the logo in `_images` in this repo (copy from https://github.com/numpy/numpy.org/tree/master/www/_static)?

**Comment by rgommers** [comment]
Thanks for doing this @mrocklin

File: index.html (hunk 1 above)
Comment (jrbourbeau) [reply index.html]: I've added `_images/quansight.svg` and replaced the image `src=` accordingly

**Comment by jrbourbeau** [comment]
Thanks for this @mrocklin!

**Pull request merged**